.gitignore

```diff
@@ -7,6 +7,9 @@ config/crd/external-crds/cns.vmware.com_*
 !config/crd/external-crds/cns.vmware.com_storagepolicyquotas.yaml
 !config/crd/external-crds/cns.vmware.com_storagepolicyusages.yaml
 
+# Do not inlcude GOSC VM mutation.
+webhooks/virtualmachine/mutation/gosc_*
```

Comment:
During the internal cayman build process, we run `git status` to list all the files added/updated by WCP patcher. The gosc_ files are part of them listed there.